packages/server/src/model-view.ts

```diff
@@ -22,6 +22,8 @@ export function view<T>(
           model,
           model.subscribe(() => {
             isInvalid = true
+            clearTimeout(updateSchedule)
+            updateSchedule = setTimeout(get, 1)
```

Comment:
why `1` tho? curious if you tried just `setTimeout(get)` and didn't work?

Reply:
it is the same behavior

Reply:
according to TS "When delay is larger than 2147483647 or less than 1, the delay will be set to 1. Non-integer delays are truncated to an integer."

Reply:
gotcha, definitely wasn't aware of that behavior!